www/index.js

```diff
@@ -21,7 +22,14 @@ document.addEventListener('keydown', (ev) => {
     case 'ArrowRight':
       curInput.right = true;
       break;
+    case 'q':
```

Comment:
just checked, and 'q' is separate from 'Q'

so we should prob handle both here